reclama/templates/index.html

```diff
@@ -14,4 +14,16 @@
       </div>
     </a>
   {% endfor %}
+
+  <div class="archived">Archived events</div>
```

Comment:
I would prefer this to be initially hidden and have a button to show last archived events. But we can skip this for now.